vendor/grammars/PHP-Twig.tmbundle

```diff
@@ -1 +1 @@
-Subproject commit 9e802d525ed87a90ec7ff842f44c0c992fd56521
+Subproject commit 77def406d70b90dff33d006478198b729e23d22c
```

Comment:
Sorry for being a pain but one of the PHP bundles causes test failures in our HTML pipeline tests on GitHub. I'm afraid I've not had a chance to investigate this yet but for now I've been skipping updating`PHP-Twig.tmbundle` and `php.tmbundle` for now. Any chance we can revert these two?